openedx/core/djangoapps/edx_global_analytics/tests/test_acceptor_api_usage.py

```diff
@@ -5,169 +5,61 @@
-        )
-
-    @patch('openedx.core.djangoapps.edx_global_analytics.models.AccessTokensStorage.objects.first')
-    def test_refreshing_token(self, mock_access_tokens_storage_model_objects_first, mock_request):
-        """
-        Verify that access_token_authorization method gets refreshed access token after authorization request to
-        acceptor if Acceptor for any reason does not has a token from edX installation in own database.
-        """
-        mock_access_token = uuid.uuid4().hex
-        mock_refreshed_access_token = uuid.uuid4().hex
-
-        mock_request.status_code = 401
-
-        mock_request.return_value.json.return_value = {
-            'refreshed_access_token': mock_refreshed_access_token
-        }
-
-        access_token_authorization(mock_access_token, 'https://mock-url.com')
-
-        mock_access_tokens_storage_model_objects_first.access_token = mock_refreshed_access_token
-
-        self.assertEqual(mock_refreshed_access_token, mock_access_tokens_storage_model_objects_first.access_token)
-        mock_access_tokens_storage_model_objects_first.save.assert_called_once()
-
-    def test_sending_statistics(self, mock_request):
-        """
-        Verify that send_instance_statistics_to_acceptor sends request to acceptor API for token dispatch statistics.
-        """
+        # Verify that send_instance_statistics_to_acceptor sends request to acceptor API for token dispatch statistics.
         send_instance_statistics_to_acceptor('https://mock-url.com', {'mock_data': 'mock_data', })
 
-        mock_request.assert_called_once_with(
-            'https://mock-url.com' + '/api/installation/statistics/', {'mock_data': 'mock_data', }
-        )
-
-    @patch('openedx.core.djangoapps.edx_global_analytics.utils.logging.Logger.info')
-    def test_successfully_sent_statistics(self, mock_logging, mock_request):
-        """
-        Verify that send_instance_statistics_to_acceptor successfully dispatches
-        installation_statistics to acceptor API.
-        """
-        mock_request.return_value.status_code = 201
-
-        send_instance_statistics_to_acceptor('https://mock-url.com', {'mock_data': 'mock_data', })
+        expected_calls = [
+            call('https://mock-url.com/api/token/registration/'),
+            call('https://mock-url.com/api/token/authorization/', data={'access_token': mock_access_token}),
+            call('https://mock-url.com/api/installation/statistics/', {'mock_data': 'mock_data'}),
+        ]
 
-        mock_logging.assert_called_with('Data were successfully transferred to OLGA acceptor. Status code is 201.')
+        self.assertEqual(mock_request.call_args_list, expected_calls)
 
     @patch('openedx.core.djangoapps.edx_global_analytics.utils.logging.Logger.info')
-    def test_unsuccessfully_sent_statistics(self, mock_logging, mock_request):
+    @data(
+        [201, 'Data were successfully transferred to OLGA acceptor. Status code is {0}.'],
```

Comment:
```py
from rest_framework import status
```
please use rest framework constants instead of numbers, e.g.
`status.HTTP_201_CREATED`